src/test/index.tsx

```diff
@@ -46,9 +51,19 @@ export function TestableWidget(props: PropsWithChildren<TestableWidgetProps>) {
             <ReduxProvider store={store}>
               <AtomProvider initialValues={props.initialAtomValues}>
                 <Web3Provider provider={hardhat.provider}>
-                  <BlockNumberProvider>
-                    <TokenListProvider list={TokenList.tokens}>{props.children}</TokenListProvider>
-                  </BlockNumberProvider>
+                  <ConnectorsProvider
```

Comment:
what's this for 🤔

Reply:
this is just to avoid an invariant failing in the test environment - i added the first test that runs code using the ConnectorsProvider